src/utils/Stylesheet.ts

```diff
@@ -1,10 +1,11 @@
 import "@fontsource/inter/500.css";
-export const PRIMARY_COLOR="#F0E21E";
-export const SECONDARY_COLOR="#F0F0F0";
-export const TERTIARY_COLOR="#7C7C7C"; 
-export const TEXT_COLOR="#000000";
-export const BACKGROUND_COLOR="#FFFFFF";
-export const ERROR_COLOR="#E32626"
-export const SUCCESS_COLOR="#20C92B"
+import "@fontsource/inter/800.css";
+export const PRIMARY_COLOR = "#F0E21E";
+export const SECONDARY_COLOR = "#F0F0F0";
+export const TERTIARY_COLOR = "#7C7C7C";
+export const TEXT_COLOR = "#000000";
+export const BACKGROUND_COLOR = "#FFFFFF";
+export const ERROR_COLOR = "#E32626";
+export const SUCCESS_COLOR = "#20C92B";
 export const FONT_FAMILY = "'Inter', sans-serif";
-export const BORDER_RADIUS= "10px";
\ No newline at end of file
+export const BORDER_RADIUS = "10px";
```

Comment:
Acá lo que yo hice fue generar una nueva variable con una fuente normal y otra en negrita

```suggestion
import "@fontsource/inter/500.css";
import "@fontsource/inter/700.css";
export const PRIMARY_COLOR = "#F0E21E";
export const SECONDARY_COLOR = "#F0F0F0";
export const TERTIARY_COLOR = "#7C7C7C";
export const TEXT_COLOR = "#000000";
export const BACKGROUND_COLOR = "#FFFFFF";
export const ERROR_COLOR = "#E32626";
export const SUCCESS_COLOR = "#20C92B";
export const FONT_FAMILY = "'Inter', sans-serif";
export const BORDER_RADIUS = "10px";
export const FONT_WEIGHT_NORMAL = "500";
export const FONT_WEIGHT_BOLD = "700";
```


En mi caso usé la de 700, fijate cual te parece que queda mejor y avisame asi ponemos las dos la misma

Reply:
sii, me gusta con 700